pkg/agent/openflow/client.go

```diff
@@ -743,21 +744,34 @@ func (c *client) Initialize(roundInfo types.RoundInfo, nodeConfig *config.NodeCo
 }
 
 func (c *client) InstallExternalFlows() error {
-	nodeIP := c.nodeConfig.NodeIPAddr.IP
-	localGatewayMAC := c.nodeConfig.GatewayConfig.MAC
+	install := func(nodeIP net.IP, isIPv6 bool) error {
+		localGatewayMAC := c.nodeConfig.GatewayConfig.MAC
 
-	var flows []binding.Flow
-	if c.nodeConfig.PodIPv4CIDR != nil {
-		flows = c.externalFlows(nodeIP, *c.nodeConfig.PodIPv4CIDR, localGatewayMAC)
-	}
-	if c.nodeConfig.PodIPv6CIDR != nil {
-		flows = append(flows, c.externalFlows(nodeIP, *c.nodeConfig.PodIPv6CIDR, localGatewayMAC)...)
+		var flows []binding.Flow
+		if !isIPv6 && c.nodeConfig.PodIPv4CIDR != nil {
```

Comment:
The check is duplicate. The function's second parameter should be `podCIDR` to avoid another stack check.

Reply:
Updated.